app/assets/javascripts/miq_application.js

```diff
@@ -320,7 +320,10 @@ function miqCheckForChanges() {
     }
   }
   // use default browser reaction for onclick
-  return true;
+  if(obj && (href = obj.getAttribute("href")))
```

Comment:
Not sure what's the reason for this..? Since we're returing true, wouldn't the default handler for href get triggered anyway?

Mostly I'm worried about it interacting with UJS / RJS, but in practice it might not matter..